### PR TITLE
Brake State -> Master

### PR DESCRIPTION
**Pull request by drorta**

### CODE REVIEW

File: src/main/java/frc/robot/RobotManager.java [hunk 1]
```diff
@@ -28,10 +29,19 @@ public class RobotManager extends LoggedRobot {
 	public void robotInit() {
 		LoggerFactory.initializeLogger();
 		BatteryUtils.scheduleLimiter();
-
```
Comment (YHerm) [inline src/main/java/frc/robot/RobotManager.java:31]: return it

File: src/main/java/frc/utils/GBSubsystem.java [hunk 2]
```diff
@@ -29,4 +31,8 @@ public final void periodic() {
 
 	protected abstract void subsystemPeriodic();
 
+	public abstract void setBreak();
+
+	public abstract void setCoast();
```
Comment (YHerm) [inline src/main/java/frc/utils/GBSubsystem.java:36]: public abstract void setBrake(boolean brake)

File: src/main/java/frc/utils/breakState/breakStateManager.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	public static void breakState() {
+		for (GBSubsystem subsystem : subsystemsList) {
+			subsystem.setBreak();
+		}
+	}
+
+	public static void coastState() {
+		for (GBSubsystem subsystem : subsystemsList) {
+			subsystem.setCoast();
+		}
```
Comment (YHerm) [inline src/main/java/frc/utils/breakState/breakStateManager.java:24]: better names please

File: src/main/java/frc/utils/breakState/breakStateManager.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package frc.utils.breakState;
+
+import frc.utils.GBSubsystem;
+
+import java.util.ArrayList;
+
+public class breakStateManager {
+
+	private static ArrayList<GBSubsystem> subsystemsList = new ArrayList<>();
```
Comment (YHerm) [inline src/main/java/frc/utils/breakState/breakStateManager.java:9]: why type name

File: src/main/java/frc/utils/breakState/breakStateManager.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package frc.utils.breakState;
+
+import frc.utils.GBSubsystem;
+
+import java.util.ArrayList;
+
+public class breakStateManager {
```
Comment (YHerm) [inline src/main/java/frc/utils/breakState/breakStateManager.java:7]:
```suggestion
public class BreakStateManager {
```

**Review by YHerm — CHANGES_REQUESTED** [review]

File: src/main/java/frc/utils/breakState/breakStateManager.java (hunk 4 above)
Comment (drorta) [reply src/main/java/frc/utils/breakState/breakStateManager.java:9]: what namee

**Pull request closed without merging**